libcxx/include/__math/traits.h

```diff
@@ -137,6 +137,18 @@ _LIBCPP_NODISCARD _LIBCPP_CONSTEXPR_SINCE_CXX23 _LIBCPP_HIDE_FROM_ABI bool isnor
   return __x != 0;
 }
 
```

Comment:
The `is_floating_point` constrained overload is now unused, right? In that case it should be removed.

Reply:
Absolutely. You are right. Corrected it. I opened https://github.com/llvm/llvm-project/pull/106224 that removes the constrained overloads for `std::{isinf,isfinite,isnan}` as well.